pallets/automation-time/src/lib.rs

```diff
@@ -356,6 +299,7 @@ pub mod pallet {
 	pub type Shutdown<T: Config> = StorageValue<_, bool, ValueQuery>;
 
 	#[pallet::error]
+	#[derive(PartialEq)]
```

Comment:
was this needed for a test?

Reply:
yes, https://github.com/OAK-Foundation/OAK-blockchain/blob/71557422f5243f5acfd4c4b886425eec876a2558/pallets/automation-time/src/types.rs#L127-L131